airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/connector_runner.py

```diff
@@ -39,7 +39,8 @@ def _prepare_volumes(self, config: Optional[Mapping], state: Optional[Mapping],
         self.input_folder.mkdir(parents=True)
         self.output_folder.mkdir(parents=True)
 
-        if config:
+        # using "is not None" to allow falsey config objects like {} to still write
```

Comment:
This was a bug in the testing code that prohibited the user of empty or falsey config objects (does this count as related to the [Low-Code CDK?](https://airbyte.com/connector-contest) 💸 ).

The connector works in the UI and from the CLI regardless, this only hurts testing.